### DISCUSSION TITLE
提示无权访问

### DISCUSSION THREAD

**Discussion by Minczm** [pjy612/SteamManifestCache#749] [Question / Answer]
- 第一次没填token的时候正常使用
- 填写token出现这个报错，是否因为我的token填写有错误
![image](https://github.com/user-attachments/assets/00aefb3d-177e-41bc-989b-e25be8453982)

**Comment by pjy612** [comment]
没权限 和 Free 别填token

**Reply by Minczm** [reply]
如果没填会显示Free token was expire.是SAI没加入内测之前只能使用一次吗

**Reply by pjy612** [reply]
Free 到期没续罢了。。。

**Reply by BlakeJason** [reply]
大佬，这个free到期了，该如何续？求解答感谢

**Reply by pjy612** [reply]
> 大佬，这个free到期了，该如何续？求解答感谢

有人分享 就续
